src/main/java/lotto/validation/LottoNumberValidator.java

```diff
@@ -0,0 +1,31 @@
+package lotto.validation;
+
+import java.util.HashSet;
+import java.util.List;
+
+public class LottoNumberValidator {
+
+    private static final int LOTTO_SIZE = 6;
+
+    private LottoNumberValidator() {
+    }
+
+    public static void validate(List<Integer> numbers) {
+        validateSize(numbers);
+        validateDuplication(numbers);
+    }
+
+    private static void validateSize(List<Integer> numbers) {
+        if (numbers.size() != LOTTO_SIZE) {
+            throw new IllegalArgumentException("로또 번호는 6개여야 합니다.");
+        }
+    }
+
+    public static void validateDuplication(List<Integer> numbers) {
+
+        int deduplicatedNumbersCount = new HashSet<>(numbers).size();
```

Comment:
```suggestion
        int uniqueNumbersCount = new HashSet<>(numbers).size();
```
변수명을 조금 더 알아보기 쉽게 작성해 봅시다!

Reply:
조언 감사드립니다! 변수 네이밍에도 좀 더 신경을 써야겠습니다!